README.md

```diff
@@ -2,36 +2,39 @@
 
 API for Node & Client
 
+## Setup [![pyVersion27](https://img.shields.io/badge/python-2.7-blue.svg)](https://www.python.org/download/releases/2.7/)
 
+- Setup `settings.py` file:
```

Comment:
If all variables are read from environment, usually configuring them in files like `settings.py` is not need anymore

Reply:
Starting development environment should be be easy as:
```
git clone ...
docker-compose up
```